api/CanvasRenderingContext2D.json

```
@@ -409,7 +409,8 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "90"
+              "version_added": "90",
+              "notes": "The gradient starts from a line going vertically up from the center, like the equivalent CSS function."
```

I think we should make it clear that there is a different implementation going on. Then, when fixed in a later version, we can have a new range that used version_added without partial implementation. 
```suggestion
              "notes": "Implements an older version of the specification. The gradient starts from a line going vertically up from the center, like the equivalent CSS function.",
              "partial_implementation": true
```